lib/file_scope.rb

```diff
@@ -16,6 +16,13 @@
 # you may find current contact information at www.suse.com
 
 class FileScope < Machinery::Object
+  def write_files_tarball(destination)
+    FileUtils.cp(
+      File.join(scope_file_store.path, "files.tgz"),
+      destination
+    )
+  end
```

Comment:
Right, we were also pondering about that. But wrinting a tarball doesn't make sense for `SystemFile`s of type "file", because all of them are encapsulated in only one tarball.

We put it in the scope itself because the result will be that one tarball with all files of that scope (and links, but not directories) will be written somewhere.